packages/main/src/themes/CardHeader.css

```diff
@@ -43,6 +43,16 @@
 	pointer-events: none;
 }
 
+.ui5-card-header-first-line {
```

Comment:
"first line" implies that there is a "second line" too, but there isn't a "second line" class in the code.

Can you think of a different name for this class?